README.md

```diff
@@ -10,14 +10,12 @@ Author: Bridget Hornsby
 
```

Comment:
The theme URI is misspelled. It does not show here, but it has uct instead of utc in the URI.

Reply:
Doesn't show on my local either. Though the versioning does.

Reply:
This is corrected and merged into develop: https://github.com/UTCWeb/utc-tailwind-genesis-theme/pull/22